examples/titanic/simple_model_training.py

```diff
@@ -19,14 +19,15 @@
 # Define Ludwig model object that drive model training
 model = LudwigModel(config='./model1_config.yaml',
                     logging_level=logging.INFO)
+dataset = Titanic().load()
```

Comment:
Let's use the convenience function:

```
from ludwig.datasets import titanic

dataset = titanic.load()
```

Reply:
added with the split, see the latest code